libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -197,15 +189,20 @@ - (void)initializeAppViewState
     self.window.rootViewController = self.viewController;
     [self.window makeKeyAndVisible];
 }
-                                                            
 
-@end
+- (void)logWithLevel:(DDLogLevel)level format:(NSString *)format, ... {
```

Comment:
I would go a step further, and suggest that we implement an `[SFSDKLogger sharedInstance]` method, that's basically a convenience to `[SFSDKLogger sharedInstanceWithComponent:@"default"]`.  Thoughts?